OMPython/OMParser/__init__.py

```diff
@@ -41,36 +41,28 @@
 next_set = []
 next_set.append('')
 
+def bool_from_string(string):
+    if string in {'true', 'True', 'TRUE'}:
```

Comment:
I also thought about that, but that would actually do something different from the current implementation, which only checks for those particular strings. This change was meant to be a pure refactor.